docs/index.md

```diff
@@ -117,6 +117,7 @@ Our documentation is organized into the following categories:
   * [MI250](./conceptual/gpu-arch/mi250.md)
   * [MI300](./conceptual/gpu-arch/mi300.md)
 * [GPU memory](./conceptual/gpu-memory.md)
+* [Setting the number of CUs](./conceptual/setting-cus)
```

Comment:
Adding this to Conceptual might not be correct - the title makes this look like procedure (with the gerund "Setting").
If this is a procedure, it should be in the How to section. 
If conceptual, it should be renamed. 

![image](https://github.com/ROCm/ROCm/assets/164940351/e885c2fd-831b-488a-a9b9-b81679b20750)

Reply:
Moved to how-to